apps/maeilmail/src/components/myQuestionList/apis/getMyQuestions.ts

```diff
@@ -1,14 +1,22 @@
 import mainClient from '@/common/apis/client/mainClient';
 import { pathGenerator } from '@/common/apis/constants/routes';
-import { QuestionCategoryEN } from '@/common/types/question';
+import { MyQuestion, QuestionCategoryEN } from '@/common/types/question';
+
+interface MyQuestionsResponse {
```

Comment:
사소한 부분인데, Response 타입 정의할 때 네이밍을 `HTTP Method + Data명 + Response` 형식으로 해왔어서, `GetMyQuestionsResponse`로 수정해주실 수 있을까요?

```suggestion
interface GetMyQuestionsResponse {
```

Reply:
알겠습니다!